src/pages/ProposalsList/ProposalsList.component.tsx

```diff
@@ -33,31 +34,50 @@ const proposalsListColumns: DataTableColumn<Proposal>[] = [
 
 export const ProposalsListComponent = ({
   proposals,
-}: ProposalsListComponentProps) => (
-  <PageView
-    title={
-      <Typography variant="h2" padding={1}>
-        BABELFISH BITOCRACY
-      </Typography>
-    }
-  >
-    <DataTable
-      data={proposals}
-      columns={proposalsListColumns}
-      tableTitle="GOVERNANCE PROPOSALS"
-      tableAction={<Button variant="text">+CREATE PROPOSAL</Button>}
-      containerSx={{ minHeight: 100 }}
-    />
-
-    <Box
-      sx={{
-        pt: 3,
-        width: '100%',
-        display: 'flex',
-        justifyContent: 'center',
-      }}
+}: ProposalsListComponentProps) => {
+  const [isAddDialogOpen, setIsAddDialogOpen] = useState(false);
+  return (
+    <PageView
+      title={
+        <Typography variant="h2" padding={1}>
+          BABELFISH BITOCRACY
+        </Typography>
+      }
     >
-      <Button variant="outlined">View All Proposals</Button>
-    </Box>
-  </PageView>
-);
+      <AddProposal
```

Comment:
Render here the Container of AddProposal once it's ready, remove the `onSubmit` prop since a submitFunction will be created inside it